logic/scripts/drops.py

```diff
@@ -58,7 +58,7 @@ def get_drops(allPast):
     global cached_drops
     global last_fetch
     drops = []
-    next_fetch = last_fetch + timedelta(minutes=fetch_interval_in_minutes)
+    next_fetch = last_fetch + timedelta(seconds=fetch_interval_in_minutes)
```

Comment:
should this be minutes= ?

Reply:
Yeah, good catch. I was testing... I think I'll back it off from 30 mins to 10 mins, though